create-rust-app/src/mailer.rs

```diff
@@ -29,9 +53,17 @@ pub struct Mailer {
     ///
     /// set by the `SEND_MAIL` environment variable
     pub actually_send: bool,
+    #[cfg(feature = "plugin_auth")]
+    // Structure containing email templates to be used for various purposes
+    pub templates: Box<dyn EmailTemplates>,
 }
 
 impl Default for Mailer {
+    #[cfg(feature = "plugin_auth")]
+    fn default() -> Self {
+        Self::new(Box::new(DefaultMailTemplates::default()))
+    }
+    #[cfg(not(feature = "plugin_auth"))]
```

Comment:
when using the auth plugin, instantiate Mailer.templates to be the default templates